frontend/ui/otamanager.lua

```diff
@@ -220,7 +220,15 @@ function OTAManager:fetchAndProcessUpdate()
                 ),
                 ok_text = _("Update"),
                 ok_callback = function()
-                    if Device:isSDL() then
+                    local isAndroid, android = pcall(require, "android")
+                    if isAndroid then
+                        -- download the package if not present.
+                        if android.download(link, ota_package) then
+                            android.notification(T(_("File %1 is already downloaded"), ota_package))
```

Comment:
```suggestion
                            android.notification(T(_("The file %1 already exists."), ota_package))
```

Either "already exists" or "has already been downloaded".

"The file %1 already exists" is already part of our translation memory.